source/slang/slang-capabilities.capdef

```diff
@@ -397,3 +451,80 @@ alias DX_6_4 = sm_6_4;
+alias sm_2_0_GLSL_400 = sm_4_0 | glsl | spirv_1_0 | cuda | cpp;
+alias appendstructuredbuffer = sm_5_0 + shader_stages_fragment_compute;
+alias atomic_hlsl = _sm_4_0;
+alias atomic_hlsl_nvapi = _sm_4_0 + hlsl_nvapi;
+alias atomic_hlsl_sm_6_6 = _sm_6_6;
+alias byteaddressbuffer = sm_4_0;
+alias byteaddressbuffer_rw = sm_4_0 + shader_stages_fragment_compute;
+alias consumestructuredbuffer = sm_5_0 + shader_stages_fragment_compute;
+alias fragmentprocessing = fragment + sm_5_0;
+alias fragmentprocessing_derivativecontrol = fragment + sm_5_0 + GL_ARB_derivative_control;
+alias getattributeatvertex = fragment + _sm_6_1 | fragment + GL_EXT_fragment_shader_barycentric;
+alias memorybarrier_compute = compute + sm_5_0;
+alias structuredbuffer = sm_4_0;
+alias structuredbuffer_rw = sm_4_0 + shader_stages_fragment_compute;
+alias texture_sm_4_1 = sm_4_1;
+alias texture_sm_4_1_fragment = sm_4_1 + fragment;
+alias texture_sm_4_1_clamp_fragment = texture_sm_4_1_fragment;
+alias texture_sm_4_1_vertex_fragment_geometry = sm_4_1 + shader_stages_vertex_fragment_geometry;
+
+alias atomic_glsl_float1 = GL_EXT_shader_atomic_float;
+alias atomic_glsl_float2 = GL_EXT_shader_atomic_float2;
+alias atomic_glsl_halfvec = GL_NV_shader_atomic_fp16_vector;
+alias atomic_glsl = GLSL_430_SPIRV_1_0;
+alias atomic_glsl_int64 = atomic_glsl + GL_EXT_shader_atomic_int64;
+alias GLSL_430_SPIRV_1_0_compute = GLSL_420_SPIRV_1_0 + compute;
+alias image_samples = GLSL_450 | spirv_1_0 + spvImageQuery | sm_4_1;
+alias image_query = GLSL_430 | GLSL_430 + spvImageQuery;
+alias image_loadstore = GL_EXT_shader_image_load_store + GLSL_420;
+alias nonuniformqualifier = sm_5_1;
+alias printf = GL_EXT_debug_printf | _sm_4_0 | _cuda_sm_2_0 | cpp;
+alias texturefootprint = GL_NV_shader_texture_footprint + GLSL_450 | hlsl_nvapi + _sm_4_0;
+alias texturefootprintclamp = texturefootprint + GL_ARB_sparse_texture_clamp;
+alias texture_shadowlod = GL_EXT_texture_shadow_lod + GLSL_140;
+alias texture_cube = GLSL_400;
+alias shader5_sm_4_0 = GL_ARB_gpu_shader5 | sm_4_0;
+alias shader5_sm_5_0 = GL_ARB_gpu_shader5 | sm_5_0;
+
+alias atomic_glsl_hlsl_cuda_float1 = atomic_glsl_float1 | atomic_hlsl_nvapi | _cuda_sm_2_0;
+alias atomic_glsl_hlsl_cuda_float2 = atomic_glsl_float2 | atomic_hlsl_nvapi | _cuda_sm_2_0;
+alias atomic_glsl_hlsl_cuda2_int64 = atomic_glsl_int64 | atomic_hlsl_nvapi | _cuda_sm_2_0;
+alias atomic_glsl_hlsl_cuda5_int64 = atomic_glsl_int64 | atomic_hlsl_nvapi | _cuda_sm_6_0;
+alias atomic_glsl_hlsl_cuda6_int64 = atomic_glsl_int64 | atomic_hlsl_nvapi | _cuda_sm_6_0;
+alias atomic_glsl_hlsl_cuda9_int64 = atomic_glsl_int64 | atomic_hlsl_nvapi | _cuda_sm_9_0;
+
+alias breakpoint = GL_EXT_debug_printf | hlsl | _cuda_sm_8_0 | cpp;
+
+alias rayobject = raytracing | rayquery;
+alias raytracing_allstages = raytracing_stages + raytracing;
+alias raytracing_anyhit = anyhit + raytracing;
+alias raytracing_int = raytracing_stages_intersection + raytracing;
```

Comment:
Can you rename this to something more readable? like
raytracing_intersection or something?